boilerplate/jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   transformIgnorePatterns: [
     "<rootDir>/node_modules/(react-clone-referenced-element|@react-native-community|react-navigation|@react-navigation/.*|@unimodules/.*|native-base|react-native-code-push)",
   ],
-  testPathIgnorePatterns: ["<rootDir>/node_modules/", "/detox", "@react-native"],
+  testPathIgnorePatterns: ["<rootDir>/node_modules/", "<rootDir>/.maestro/", "@react-native"],
```

Comment:
Is this needed?

Reply:
As long as we don't add any `.test.{ts,js}` paths in this directory, it isn't an issue.

Maestro does let you add minimal `.js` files to execute in tests, so it may be possible that someone tries to add a `.test.js`.

My gut tells me this is good to include just as a matter of intent, making it clear that jest should not run these files